DEVELOPING.rst

```diff
@@ -176,6 +176,23 @@ you can do:
    docker exec -i -t opendatacernch_web_1 bash -c "cernopendata fixtures records -f /code/cernopendata/modules/fixtures/data/records/opera-ecc-datasets.json --mode insert"
    docker exec -i -t opendatacernch_web_1 bash -c "cernopendata fixtures records -f /code/cernopendata/modules/fixtures/data/records/opera-ed-datasets.json --mode insert"
 
+Frontend development
```

Comment:
The other sections are called "Working with records", "Working with ...", so "Working with JS/CSS" or "Working with frontend" or "Working with UI" might be more consistent.  (Also moving it before "Switching..." to be next to the other "Working..." tips and tricks.)